spec/rubocop/cop/rspec/repeated_description_spec.rb

```diff
@@ -73,4 +73,18 @@
       end
     RUBY
   end
+
+  it 'does not flag examples if metadata is different' do
```

Comment:
what if there is the same metatada? with same description? With different descriptions?
I would convert this to a context and add a couple of more examples

Reply:
Added two more tests